README.md

```diff
@@ -317,7 +289,7 @@ The `test_track_visitor.sign_up!` method tells TestTrack when a new identifier h
 test_track_visitor.sign_up!(:myapp_user_id, 2345)
 ```
 
-## Testing splits
+## Testing your split-dependent application code with RSpec
```

Comment:
Since we're doing split testing, I thought this could benefit from a little clarification.